bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -163,6 +171,8 @@ if args.randomize_template_order:
 else:
     template_ids = range(tmin, tmax)
 
+template_ids = numpy.intersect1d(unique_tids, template_ids)
```

Comment:
add comment to explain what is being done

Reply:
Ok, done